cluster-api/src/test/java/io/aiven/klaw/clusterapi/services/MetricsApiServiceTest.java

```diff
@@ -0,0 +1,67 @@
+package io.aiven.klaw.clusterapi.services;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+
+import io.aiven.klaw.clusterapi.utils.MetricsUtils;
+import java.util.Map;
+import javax.management.MBeanAttributeInfo;
+import javax.management.MBeanInfo;
+import javax.management.MBeanServerConnection;
+import javax.management.ObjectName;
+import javax.management.remote.JMXConnector;
+import org.assertj.core.api.AbstractThrowableAssert;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+class MetricsApiServiceTest {
+
+  @Mock private MetricsUtils metricsUtils;
+  @Mock private JMXConnector jmxConnector;
+  @Mock private MBeanServerConnection mBeanServerConnection;
+  @Mock private MBeanInfo mBeanInfo;
+  @InjectMocks private MetricsApiService metricsApiService;
+
+  @Test
+  void getMetrics() throws Exception {
+    String jmxUrl = "service:jmx:rmi:///jndi/rmi://localhost:9996/jmxrmi";
+    String objectName = "kafka.server:type=BrokerTopicMetrics,name=MessagesInPerSec";
+    MBeanAttributeInfo mBeanAttributeInfo =
```

Comment:
You can make these class constants too.
 String jmxUrl = "service:jmx:rmi:///jndi/rmi://localhost:9996/jmxrmi";
    String objectName = "kafka.server:type=BrokerTopicMetrics,name=MessagesInPerSec";

Reply:
I will make these changes.